src/Web/WebApp/Components/Common/Cards/EntityCard.razor.css

```
@@ -1,59 +1,61 @@
-/* Custom Card Styles */
-.entity-card-wrapper ::deep .entity-card {
+::deep .entity-card.mud-card {
```

Stylelint (17.3.0)

[error] Unexpected unknown pseudo-element selector "::deep" (selector-pseudo-element-no-unknown)

(selector-pseudo-element-no-unknown)